api/python/t4/api.py

```diff
@@ -214,7 +214,9 @@ class PackageList:
         """Display wrapper for list_packages"""
 
         def __init__(self, pkg_info):
-            self.pkg_names = [info['pkg_name'].replace(':latest', '') for info in pkg_info]
+            self.pkg_names = list(set(
```

Comment:
Please add a comment explaining that `list(set(foo))` is used to remove duplicate values from a list

Reply:
Done.